inspector2_enablement_with_awscli.sh

```diff
@@ -146,7 +146,7 @@ get_regions_list
 # Check if the current user is in the delegated admin or not
 is_da_account(){
     check_result=""
-    aws inspector2 get-delegated-admin-account 2>&1 |& grep -qs 'Invoking account is the delegated admin.'
+    aws inspector2 get-delegated-admin-account 2>&1 | grep -qs 'Invoking account is the delegated admin.'
```

Comment:
This operator pipes both standard output and standard error from the left hand side to the right hand side. This is a newer BASH feature, but in the README there is a warning that this script requires BASH 5

Reply:
Hello @daodennis-ri,

thank you for feedback. 
I did receive complaint about the script not working as expect with lower bash version, but I didn't have any issue on my side because I have bash 5. I also try it with other colleagues who didn't have any issue.

So you are saying that with bash 5, there is no need to add & after the pipe, is that correct?

regards,

Reply:
no I'm saying that `|&` is likely a newer BASH feature but also supporting BASH 3.2 might not be a bad idea